docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -221,7 +221,7 @@ The procedure below will create a 90GB data volume and 4GB metadata volume to us
         INFO[0027] Daemon has completed initialization
         INFO[0027] Docker daemon                                 commit=0a8c2e3 execdriver=native-0.2 graphdriver=devicemapper version=1.8.2
 
-    It is also possible to set the `--storage-driver` and `--storage-opt` flags in the Docker config file and start the daemon normally using the `service` or `systemd` commands.
+  It is also possible to set the `--storage-driver` and `--storage-opt` flags in the Docker config file and start the daemon normally using the `service` or `systemd` commands.
```

Comment:
This definitely changes

Before:
<img width="767" alt="screen shot 2015-11-20 at 14 06 51" src="https://cloud.githubusercontent.com/assets/13337345/11313220/37c5129e-8f90-11e5-9dc3-3ca6dbd3a31d.png">
After:
<img width="918" alt="screen shot 2015-11-20 at 14 04 53" src="https://cloud.githubusercontent.com/assets/13337345/11313222/3c6e5a9e-8f90-11e5-9470-e8feeabbb8ca.png">

I know markdown doesn't care, but in the raw source we have sections 5 and 6, and then sections 5 and 6 again.